src/io/github/sspanak/tt9/ime/modes/ModePredictive.java

```diff
@@ -163,11 +259,35 @@ public boolean setWordStem(Language language, String wordStem, boolean exact) {
 	 * getWordStem
 	 * If "setWordStem()" has accepted a new stem by returning "true", it can be obtained using this.
 	 */
+	@Override
 	public String getWordStem() {
 		return stem;
 	}
 
 
+	/**
+	 * loadStaticSuggestions
+	 * Similar to "loadSuggestions()", but loads suggestions that are not in the database.
+	 * Returns "false", when there are no static suggestions for the current digitSequence.
+	 */
+	private boolean loadStaticSuggestions() {
+		if (digitSequence.equals("0")) {
+			suggestions = Punctuation.Secondary;
+		} else if (containsOnly1.matcher(digitSequence).matches()) {
+			if (digitSequence.length() == 1) {
+				suggestions = Punctuation.Main;
+			} else {
+				digitSequence = digitSequence.length() <= maxEmojiSequence.length() ? digitSequence : maxEmojiSequence;
+				suggestions = Punctuation.getEmoji(digitSequence.length() - 2);
```

Comment:
Double check this. It may be causing #125. If not just set the composing text properly and fix the bug.